<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingDlg</class>
 <widget class="QWidget" name="SettingDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>786</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>配置信息</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>当前配置</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTreeWidget" name="treeWidget">
        <column>
         <property name="text">
          <string notr="true">键</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>值</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>说明</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="expandAll">
       <property name="text">
        <string>全部展开</string>
       </property>
       <property name="icon">
        <iconset resource="FlashTool.qrc">
         <normaloff>:/Images/Resources/Images/expand.ico</normaloff>:/Images/Resources/Images/expand.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addNode">
       <property name="text">
        <string>添加文件节点</string>
       </property>
       <property name="icon">
        <iconset resource="FlashTool.qrc">
         <normaloff>:/Images/Resources/Images/add.ico</normaloff>:/Images/Resources/Images/add.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="deleteNode">
       <property name="text">
        <string>删除文件节点</string>
       </property>
       <property name="icon">
        <iconset resource="FlashTool.qrc">
         <normaloff>:/Images/Resources/Images/del.ico</normaloff>:/Images/Resources/Images/del.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveData">
       <property name="text">
        <string>保存数据</string>
       </property>
       <property name="icon">
        <iconset resource="FlashTool.qrc">
         <normaloff>:/Images/Resources/Images/save.ico</normaloff>:/Images/Resources/Images/save.ico</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit">
       <property name="text">
        <string>退出</string>
       </property>
       <property name="icon">
        <iconset resource="FlashTool.qrc">
         <normaloff>:/Images/Resources/Images/treeExit.ico</normaloff>:/Images/Resources/Images/treeExit.ico</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FlashTool.qrc"/>
 </resources>
 <connections/>
</ui>
